I.write-tests/05-get-second-third-smallest/get-second-third.js

```diff
@@ -3,5 +3,5 @@ module.exports = function (array) {
     newArray.sort(function (x, y) {
         return x > y;
     });
-    return [newArray[1], newArray[2]];
+    return [newArray[1], newArray[2], array];
```

Comment:
newArray is an entirely new copy from slice(), so there's no need to worry about the original array changing, you only have to test for it in the test file